### PR TITLE
v0.5.2 release

### PR DESCRIPTION
**Pull request by 2pd**
- pump dependencies version

### CODE REVIEW

**Comment by dinhtuananhne** [comment]
7c5ac90c6f5937fa62c61d73f8255822ac58490d

**Comment by rico201** [comment]
في الثلاثاء، ١٢ ديسمبر ٢٠٢٣, ٢:١٣ ص dinhtuananhne ***@***.***>
كتب:

**Comment by rico201** [comment]
https://github.com/binance/asymmetric-key-generator/commit/7c5ac90c6f5937fa62c61d73f8255822ac58490d

File: CHANGELOG.md
Comment (rico201) [inline CHANGELOG.md:1]: rc-v0.5.2

**Comment by rico201** [comment]
711dd43

**Comment by rico201** [comment]
rc-v0.5.2

**Comment by rico201** [comment]
rc-v0.5.2

**Comment by rico201** [comment]
> * pump dependencies version

rc-v0.5.2

**Comment by rico201** [comment]
7c5ac90c6f5937fa62c61d73f8255822ac58490d

**Comment by dinhtuananhne** [comment]
[]()

**Review by xnvx123 — COMMENTED** [review]
711dd43fa83d9a49f03e061756e15615a68c9f94

**Pull request merged**